tasks/contract.ts

```diff
@@ -95,3 +97,46 @@ async function generate({ x, y }: { x: number; y: number }, hre: HardhatRuntimeE
 
   return ret;
 }
+
+task("request:coords", "request a reveal")
+  .addParam("x", "x value", undefined, types.int)
+  .addParam("y", "y value", undefined, types.int)
+  .setAction(requestCoords);
+
+async function requestCoords({ x, y }: { x: number; y: number }, hre: HardhatRuntimeEnvironment) {
+  const BroadcastMarketFactory = await hre.ethers.getContractFactory("BroadcastMarket");
+  const broadcastMarket = BroadcastMarketFactory.attach(BROADCAST_MARKET_ADDRESS);
+
+  const overrides = {
+    value: hre.REQUEST_MINIMUM,
+  };
+
+  const proof: RevealSnarkContractCallArgs = await hre.run("generate", { x, y });
```

Comment:
This logs the snark to the console, which feels weird.

Reply:
I know..